source_code/src/LOGIC/logic_fwflash_storage.h

```diff
@@ -274,8 +274,47 @@
     #define ID_STRING_LED3              69
     #define ID_STRING_LED4              70
 
+#ifdef ENABLE_CREDENTIAL_MANAGEMENT
```

Comment:
this makes me think... should you add 5 "empty" strings after our last current string? this way if I have to add another string I don't have to include your strings...